cfme/configure/access_control.py

```diff
@@ -1237,6 +1239,10 @@ def is_displayed(self):
 class DetailsTenantView(ConfigurationView):
     """ Details Tenant View """
     toolbar = View.nested(AccessControlToolbar)
+    name = Text('Name')
+    description = Text('Description')
+    parent = Text('Parent')
+    table = BaseListEntity()
```

Comment:
**REQUIRED**
I didn't find any usage of this `table` attribute. What's the purpose of it? Also you shouldn't use `BaseListEntity` directly, look how it's implemented in [`host_views.py`](https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/host_views.py#L73)

Reply:
This table contains info about what's the quota set and what's the current quota in use or exceeded. 
Future test cases are planned around it.